avm/utilities/pipelines/publish/Publish-ModuleFromPathToPBR.ps1

```diff
@@ -77,11 +77,11 @@ function Publish-ModuleFromPathToPBR {
   $null = Convert-TokensInFileList @tokenConfiguration
 
   # Double-check that tokens are correctly replaced
-  $templateContent = Get-Content -Path $moduleBicepFilePath
+  $templateContent = bicep build $moduleBicepFilePath --stdout
```

Comment:
Do we want to include minor changes to modules missing publishing in this PR? Or should that happen separately?

Reply:
Hey @eriqua, it's a good question. We can - or - we re-try to use the publish from tag workflow first. Thoughts? Either is fine with me.

Reply:
Unfortunately that wouldn't work. The code to publish the tag is checked out from the tag itself, so this fix wouldn't be included.

Reply:
on it

Reply:
done